### PR TITLE
AFK Timeout text fix

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request
Fixes a button in the afk timeout prompt.

## Why It's Good For The Game
To show I can't code and just take suggestions

## Changelog
Doesn't need one

## Other
I web edited this PR because I am a coding pro :tm:

### CODE REVIEW

**Comment by SpaceManiac** [comment]
I think the real problem may be here: `<span class='danger'You may reconnect via t`

**Comment by YakumoChen** [comment]
AFK Timeout gives you the message, the text I edited is the button to reconnect.

File: code/controllers/subsystem/server_maint.dm [hunk 1]
```diff
@@ -37,7 +37,7 @@ SUBSYSTEM_DEF(server_maint)
 			var/cmob = C.mob
 			if (!isnewplayer(cmob) || !SSticker.queued_players.Find(cmob))
 				log_access("AFK: [key_name(C)]")
-				to_chat(C, "<span class='userdanger'>You have been inactive for more than [DisplayTimeText(afk_period)] and have been disconnected.</span><br><span class='danger'You may reconnect via the button in the file menu or by <b><u><a href='byond://winset?command=.reconnect'>clicking here to reconnect</a></b></u></span>")
+				to_chat(C, "<span class='userdanger'>You have been inactive for more than [DisplayTimeText(afk_period)] and have been disconnected.</span><br><span class='danger'You may reconnect via the button in the file menu or by <b><u><a href='byond://winset?command=.reconnect'>Click here to reconnect</a></b></u></span>")
```
Comment (SpaceManiac) [inline code/controllers/subsystem/server_maint.dm]:
```suggestion
				to_chat(C, "<span class='userdanger'>You have been inactive for more than [DisplayTimeText(afk_period)] and have been disconnected.</span><br><span class='danger'>You may reconnect via the button in the file menu or by <b><u><a href='byond://winset?command=.reconnect'>clicking here to reconnect</a></u></b>.</span>")
```

**Comment by YakumoChen** [comment]
it took me 5 times reading it to see what you were trying to do and decided you were probably right regardless

**Comment by Angustmeta** [comment]
I think the real problem here is `clicking here to reconnect`

**Comment by Detective-Google** [comment]
I'm a little blind but I'm presuming this fixes the "clicking here to reconnect" button.

thank fuck that bothered me way more than it should.

**Comment by Tlaltecuhtli** [comment]
did you test it

**Comment by YakumoChen** [comment]
For 100 hours yes

**Pull request merged**